File: simple_geo3x3.py
import geo3x3

## get Geo3x3 code from latitude / longitude / level
code = geo3x3.encode(35.65858, 139.745433, 14)
print(code)

## get location from Geo3x3 code
pos = geo3x3.decode('E9139659937288')
print(pos)

File: simple_geo3x3.exs
Code.require_file("geo3x3.ex")

## get Geo3x3 code from latitude / longitude / level
code = Geo3x3.encode(35.65858, 139.745433, 14)
IO.inspect(code)

## get location from Geo3x3 code
pos = Geo3x3.decode("E9139659937288")
IO.inspect(pos)
